### PR TITLE
Wrap client creation in a function and recreate for tests

### PR DESCRIPTION
**Pull request by aldehir**
The commit is pretty ugly, but this should resolve the issue with imports in test files.

### CODE REVIEW

File: test/conftest.py [hunk 1]
```diff
@@ -5,17 +5,20 @@
...
+    # Recreate client with the pytest-asyncio event loop
+    client.c = client.create_client()
 
     # Always use a clean database
     db = tmp_path_factory.mktemp("pubobot_tests") / "db.sqlite3"
 
     console.init(enable_input=False)
     scheduler.init()
-    bot.init()
+    bot_.init()
```
Comment (dcramps) [inline test/conftest.py]: why is this needed?

**Review by dcramps — APPROVED** [review]

File: test/conftest.py (hunk 1 above)
Comment (aldehir) [reply test/conftest.py]: Fixture name bot conflicts with the bot module. So it's renamed to bot_ but we can rename the fixture instead if preferred.

**Comment by aldehir** [comment]
@dcramps perhaps this is preferable. Renaming the `bot` fixture to `pbot` means we can do `from pubobot import bot` in tests without conflict.

**Comment by dcramps** [comment]
`pbot` works for me. lgtm
